### ISSUE TITLE
Add ability to filter which AD groups are registered in Netbox

### ISSUE THREAD

**Issue by FlochonR** [marcus-crane/netbox-plugin-azuread#10]
Hello @marcus-crane !

I'm running the version 3.0.5 of Netbox and version 1.1.0 of your plugin with this configuration
```
PLUGINS = [
  'netbox_plugin_azuread'
]
PLUGINS_CONFIG = {
  'netbox_plugin_azuread': {
    'CLIENT_ID': 'toto,
    'CLIENT_SECRET': 'toto',
    'AUTHORITY': 'https://login.microsoftonline.com/toto/',
    'LOGIN_URL': '/login/',
    'REPLY_URL': '/complete/',
    'SCOPES': ['https://graph.microsoft.com/.default'],
    'AD_GROUP_MAP': {
      'STAFF': ['NETBOX-Users'],
      'SUPERUSER': ['NETBOX-Administrators']
    }
  }
}
REMOTE_AUTH_AUTO_CREATE_USER = True
REMOTE_AUTH_BACKEND = 'netbox_plugin_azuread.backends.AzureADRemoteUserBackend'
REMOTE_AUTH_ENABLED = True
```

The users are well created and they are on the group Azure AD NETBOX-Users or NETBOX-Administrators, the problem is the flag isn't added to the user 
Perhaps I will do something special in Azure AD config ?

Thanks for your help,
FlochonR

**Comment by qxmips** [comment]
@FlochonR please check https://github.com/marcus-crane/netbox-plugin-azuread/issues/3

**Comment by marcus-crane** [comment]
Hey @FlochonR, this issue got closed automatically a bit early but [1.2.0](https://github.com/marcus-crane/netbox-plugin-azuread/releases/tag/1.2.0) has just been released which includes the filter functionality you were after.

**Comment by FlochonR** [comment]
Hello @marcus-crane @tatsuyaueda 

Thanks for the work, it's great !!
